pygluu/kubernetes/templates/helm/gluu/charts/fido2/templates/_helpers.tpl

```diff
@@ -67,6 +67,21 @@ Create user custom defined secret envs
 {{- end }}
 {{- end }}
 
+{{/*
+Create GLUU_JAVA_OPTIONS ENV for passing custom work and detailed logs
+*/}}
+{{- define "fido2.customJavaOptions"}}
+{{ $custom := "" }}
+{{- if .Values.global.fido2.gluuCustomJavaOptions }}
```

Comment:
So this is only activated if the user passes gluuCustomJavaOptions which isn't true. We want to pass in  the required params here that would be -XX:MaxDirectMemorySize -Xmx and -DCN_IDP_HOST nevertheless.